styles.css

```
@@ -9936,3 +10036,514 @@ body:has(#side-panel.active) #close-fullscreen-cover-btn {
+/* Fullscreen layout rebuild on PR 378 base */
+#fullscreen-cover-overlay .fullscreen-shell {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    align-items: stretch;
+    justify-content: center;
+    min-height: 0;
+    overflow: hidden;
+}
+
+#fullscreen-cover-overlay .fullscreen-main-view {
```

Stylelint (17.6.0)

[error] Expected selector "#fullscreen-cover-overlay .fullscreen-main-view" to come before selector "#fullscreen-cover-overlay.ui-hidden .fullscreen-main-view", at line 4102 (no-descending-specificity)

(no-descending-specificity)